documentation/release_notes.markdown

```diff
@@ -61,179 +94,253 @@ This release retires all api versions before v4 and promotes v4 to be
+      "version" : "e4c339f",
+      "transaction-uuid" : "53b72442-3b73-11e3-94a8-1b34ef7fdc95",
+      "environment" : "production",
+      "edges" : [{"source": {"type": "package",
+                             "title": "openssh-server"}
+                  "target": {"type": "file",
+                             "title": "/etc/ssh/sshd_config"}
+                  "relationship": "before"}],
+      ...
+    }
+
+    This will now be nested with an href and the entity included as the value of the "data" key:
+
+    {
+      "certname" : "yo.delivery.puppetlabs.net",
+      "version" : "e4c339f",
+      "transaction-uuid" : "53b72442-3b73-11e3-94a8-1b34ef7fdc95",
+      "environment" : "production",
+      "edges" : {"href": "/pdb/query/v4/catalogs/yo.delivery.puppetlabs.net/edges",
+                 "data": [{"source": {"type": "package",
+                           "title": "openssh-server"}
+                           "target": {"type": "file",
+                           "title": "/etc/ssh/sshd_config"}
+                  "relationship": "before"}],
+      ...
+    }
+
+    Note that HSQLDB users will not have the `data` keys prepopulated
+    and will need to follow the `href` included in the response to get the
+    nested entity data. PostgreSQL users will always have the nested
+    entity included.
+
+    The cause of this change is a new feature in PostgreSQL that
+    allows us to improve the performance of queries that contain this
+    nested entity data. HSQLDB does not have an analogous feature,
+    which required us to formalize how this nested entity data is
+    returned.
+
+    For PostgreSQL API consumers this means, the extra `data` key needs to be
+    traversed to access the real related object data.
+
+* Add a `noop` flag to reports ([PDB-1177](https://tickets.puppetlabs.com/browse/PDB-1177))
+
+    This adds a `noop` field to the reports object, which is a
+    required boolean flag indicating whether the run producing the
+    report used `--noop` (for example: `puppet agent -t --noop`).
 
-* Change "name" key to "certname" ([PDB-1099](https://tickets.puppetlabs.com/browse/PDB-1099))
+* Remove `com.` prefix from JMX and ActiveMQ endpoints ([PDB-863](https://tickets.puppetlabs.com/browse/PDB-863))
 
-    For clarity and consistency across endpoints, the "name" key has
-    been changed to "certname" where appropriate. This affects
-    responses from the /pdb/query/v4/catalogs endpoint, queries to it, and the
-    "replace catalog" and "replace facts" commands.
+    The `com.` prefix has been removed from the JMX and ActiveMQ endpoints. They are now in the
+    `puppetlabs.puppetdb` namespace. Any monitoring applications you have that
+    expect the old namespace will need to be changed.
 
-* Expanded data format is now more formalized. ([PDB-1228](https://tickets.puppetlabs.com/browse/PDB-1228))
+* The endpoints `aggregate-event-counts` and `event-counts` have now been marked as experimental.
```

Comment:
possibly an exaggeration, but I think in the context of rolling release notes the word "now" is not ever necessary